proto/cosmos/cosmos.proto

```diff
@@ -94,3 +95,50 @@ message TxData {
 
   repeated MsgData data = 1;
 }
+
+// TxResponse defines a structure containing relevant tx data and metadata. The
+// tags are stringified and the log is JSON decoded.
+message TxResponse {
+  option (gogoproto.goproto_getters) = false;
+
+  int64 height = 1;
+  string txhash = 2 [(gogoproto.customname) = "TxHash"];
+  string codespace = 3;
+  uint32 code = 4;
+  string data = 5;
+  string raw_log = 6;
+  repeated ABCIMessageLog logs = 7 [(gogoproto.castrepeated) = "ABCIMessageLogs", (gogoproto.nullable) = false];
+  string info = 8;
+  int64 gas_wanted = 9;
+  int64 gas_used = 10;
+  google.protobuf.Any tx = 11;
+  string timestamp = 12;
+}
```

Comment:
I do want to note that I do think this stuff should live in the root `cosmos` proto package. I'm not sure where it should go but I don't feel comfortable with it here. Any thoughts? I'd also like to take out the other `Result`, `SimulationResponse`, etc. stuff and leave just `Coin`, `DecCoin` & `Int/DecProto` preferably before 0.40 is done.

Reply:
> Is the `Marshal/UnmarshalAny` needed for this PR?

Nope, not necessary for this PR, just flagging that I want to make sure it is covered somewhere to be tackled in the #6213 bullet list if we want to include it in 0.40

Reply:
Great 👍